### PR TITLE
Setup project with linters 

### PR DESCRIPTION
**Pull request by ankitt26**
In this branch, I did -

- Create React app 
- Install react-redux 
- Install react logger 
- Install react-router 
- Setup liters
- Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,199 @@
...
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ankitt26/Math-magicians/issues).
+
```
Comment (sunga12) [inline README.md:174]: Kindly make sure the Issues link is the correct one

File: README.md [hunk 2]
```diff
@@ -1 +1,199 @@
...
+> I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/ankitt26/Math-magicians/blob/develop/LICENSE) licensed.
```
Comment (sunga12) [inline README.md:197]: Kindly make sure the License link is the correct one

**Review by sunga12 — CHANGES_REQUESTED** [review]
Hi @ankitt26 



Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!


## Highlights
✔️ Setup Complete
✔️ All linter passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sunga12 — APPROVED** [review]
Hi @ankitt26  

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ankitt26** [comment]
Thanks for review 😊.

**Pull request merged**